components/x-live-blog-wrapper/src/LiveBlogWrapper.jsx

```diff
@@ -4,9 +4,13 @@ import { withActions } from '@financial-times/x-interaction'
 import { listenToLiveBlogEvents } from './LiveEventListener'
 
 const withLiveBlogWrapperActions = withActions({
-	insertPost(post) {
+	insertPost(newPost) {
 		return (props) => {
-			props.posts.unshift(post)
+			// Check if the new post already exists in the page, so we don't end up with duplicates.
+			const newPostExists = props.posts.find((existingPost) => existingPost.id === newPost.id)
```

Comment:
I have seen `post.postId` and `post.id` used in this codebase, which is correct?

Reply:
I think this is a wordpress to spark cross over thing. I can't be certain if each case is using the right one without reviewing them all but I believe its that `postId` was a wordpress thing and `id` is Spark.

There might be a reason why we didn't standardise them but I am not sure 😬

Reply:
I'm going to update to `postId` for now since it seems the most current thing:

https://github.com/Financial-Times/x-dash/commit/be96eb0e44b6b0ef70a3112716fc768b302daf36